source/routing/tests/isochron_test.cpp

```diff
@@ -96,3 +107,49 @@ BOOST_AUTO_TEST_CASE(circle_test) {
     BOOST_CHECK_CLOSE(r_30, 30, 0.5);
     BOOST_CHECK_CLOSE(r_459, 459, 0.5);
 }
+
+
+BOOST_AUTO_TEST_CASE(build_ischron_test) {
+    using coord = navitia::type::GeographicalCoord;
+    coord coord_Paris = {2.3522219000000177, 48.856614};
+    coord coord_Notre_Dame = {2.35, 48.853};
+    coord coord_Rennes = {-1.68333,48.083328};
+    coord coord_Pantheon = {2.3461,48.8463};
+    coord coord_Concorde = {2.32,48.87};
+    coord coord_Luxembourg = {2.339981,48.845404};
+    double speed = 0.8;
```

Comment:
If you don't pass it in param, as you only use it in the last boost_check it would be nice to move it there